dev/dags/simple_dag_async.py

```diff
@@ -24,7 +24,7 @@
 simple_dag_async = DbtDag(
     # dbt/cosmos-specific parameters
     project_config=ProjectConfig(
-        DBT_ROOT_PATH / "original_jaffle_shop",
+        DBT_ROOT_PATH / "jaffle_shop",
```

Comment:
Curious, why are we changing the project?

Reply:
I observed we have the same project across all our example DAGs, so altered it for consistency.

Reply:
Look like `jaffle_shop` does not bigquery profile which may require here? https://github.com/astronomer/astronomer-cosmos/blob/main/dev/dags/dbt/jaffle_shop/profiles.yml

Reply:
ohh it uses profile mapping

Reply:
based on the discussions, I reverted the project to use `original_jaffle_shop`. We can work on it in a separate PR to get rid of original_jaffle_shop.